src/System.Runtime.Extensions/src/System/IO/Path.Windows.cs

```diff
@@ -93,6 +84,15 @@ public static string GetFullPath(string path)
 
             return fullPath;
         }
+        
+        public static char[] GetInvalidFileNameChars() => new char[]
```

Comment:
@stephentoub 

> Moving stuff around in the file makes it very difficult to see in the diff view if there are actually meaningful changes

Ok, I see your point. I'll move this method to the place where the field originally was then.